distributions/raspijamulus.sh

```diff
@@ -6,11 +6,11 @@ NCORES=$(nproc)
 
 # install required packages
 pkgs='alsamixergui build-essential qtbase5-dev qttools5-dev-tools libasound2-dev cmake libglib2.0-dev'
-if ! dpkg -s $pkgs >/dev/null 2>&1; then
+if ! dpkg -s "${pkgs}" >/dev/null 2>&1; then
```

Comment:
This doesn't work either. Same comment as on line 13.

Reply:
Why do we even ask?